rubygems/rubygems.org
# Refactor the mailer content to specify the MFA implementation used (TOTP)

## What problem are you solving?
Closes https://github.com/Shopify/ruby-dependency-security/issues/331
Refactored the email content to specify the MFA implementation used (TOTP)

## What should reviewers focus on?
Not the biggest fan of the lengthiness of some of the content:

ie. the email subject
> TOTP multi-factor authentication disabled on RubyGems.org

and the content body
> (TOTP (Time-Based One-Time Password) multi-factor authentication)

but also just didn't like the way TOTP MFA looked (acronym overload), but I would be okay to change some/both of these if there any suggestions 😄

## Testing 🎩
Borrowing the testing instructions from a [previous PR](https://github.com/rubygems/rubygems.org/pull/3779) within the same section of code  (thanks @garyhtou):
1. Setup OTP MFA at http://localhost:3000/settings/edit
2. Go to http://localhost:3000/letter_opener and verify MFA Enabled mailer was sent
3. Disable OTP MFA at http://localhost:3000/settings/edit
4. Go to http://localhost:3000/letter_opener and verify MFA Disabled mailer was sent
---
Can also view the updated mailers individually through http://localhost:3000/rails/mailers
- Select "totp_disabled" / "totp_enabled"
<p align="center">
  <kbd>
    <img src="https://github.com/rubygems/rubygems.org/assets/20158582/6d2453f9-fbb8-460b-ae12-2ba616c2c7d3" width=300>
    <img src="https://github.com/rubygems/rubygems.org/assets/20158582/c5700f20-21ad-4954-be87-bbbe4e78fd13" width=300>
  </kbd>
</p>
